exercise_10/index.js

```diff
@@ -0,0 +1,7 @@
+import transformCollectionToString from "./transformCollectionToString.js";
+
+let collectionNotEmpty = ["apple", "grapes", "strawberries"];
+console.log(transformCollectionToString(collectionNotEmpty));
```

Comment:
One line, one thing.

```suggestion
const output = transformCollectionToString(collectionNotEmpty) 
console.log(output);
```